src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -87,16 +86,12 @@ const DiscoverySearch = ({
 
   const handleChangeText = (text) => {
     setSearchText(text);
-    setIsTextAvailable(text.length > 0);
     debounceChangeText(text);
   };
 
   const handleOnClearText = () => {
     setSearchText('');
     setLastSearch('');
-    // setIsTextAvailable(false)
-    // debounced.cancel()
-    // discoverySearchBarRef.current.focus()
   };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The state variable `isTextAvailable` and its related logic have been removed. If this state was used elsewhere in the component or had side effects, make sure those are addressed.
```diff
- setIsTextAvailable(text.length > 0);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->